packages/embed/src/utils/get-transferable-query-params.spec.ts

```diff
@@ -0,0 +1,40 @@
+import { getTransferableQueryParams } from './get-transferable-query-params'
+
+describe('transferUrlParametersToQueryStrings', () => {
+  const location: Location = window.location
+
+  beforeAll(() => {
+    delete (window as any).location
+
+    window.location = {
+      search: '?foo=jason&bar=rachel&utm_medium=cpc&utm_campaign=camp2008&utm_source=instagram&embed-hide-footer=false',
+      href:
+        'http://localhost?foo=jason&bar=rachel&utm_medium=cpc&utm_campaign=camp2008&utm_source=instagram&embed-hide-footer=false',
+    } as any
```

Comment:
I suggest to define the query string just once, I think it increases the readability since it is quite long:

```suggestion
    const search = ''?foo=jason&bar=rachel&utm_medium=cpc&utm_campaign=camp2008&utm_source=instagram&embed-hide-footer=false''

    window.location = {
      search,
      href: 'http://localhost`${search},
    } as any
```